SimPEG/electromagnetics/time_domain/sources.py

```diff
@@ -520,6 +520,7 @@ class LineCurrent(BaseTDEMSrc):
     loc = deprecate_property(
         location, "loc", new_name="location", removal_version="0.15.0"
     )
+    current = properties.Float("current in the line", default=1.0)
```

Comment:
Is this in the unit of Amperes? If so, maybe clearly specify as such?

Reply:
there aren't any explicit units anywhere in SimPEG, as far as I can tell-- it's really just a scale for the LineCurrent